src/pybind/mgr/cephadm/tests/fixtures.py

```diff
@@ -27,17 +28,41 @@ def get_store_prefix(self, prefix):
         if k.startswith(prefix)
     }
 
+
 def get_ceph_option(_, key):
     return __file__
 
+
+def _run_cephadm(ret):
+    def foo(*args, **kwargs):
+        return ret, '', 0
+    return foo
+
+
+def match_glob(val, pat):
+    ok = fnmatch.fnmatchcase(val, pat)
+    if not ok:
+        assert pat in val
+
+
+def mon_command(*args, **kwargs):
+    return 0, '', ''
+
```

Comment:
also,
```
def no_op(*args, **kwargs):
    pass
```
so that all of the no-op mock.patches below don't need to be passed as arguments at all!

Reply:
not sure what you mean with that exactly. Could you elaborate on an example?

Reply:
in any remaining cases where the function is mocked out to do nothing but the handle isn't used in the test to force a return value or check anything then you can use this and drop the argument to the test function.